# plan manager parameter (plan_manager/)
max_vel: 2.0
max_acc: 3.0
max_jerk: 4.0
feasibility_tolerance: 0.05
control_points_distance: 0.4
use_distinctive_trajs: true
local_time_horizon: 5.0